templates/base.html

```diff
@@ -243,4 +244,248 @@
+                        <i class="fab fa-mastodon mr-1"></i> Mastodon
+                    </a>
+                    <a href="#" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fas fa-cloud mr-1"></i> Bluesky
+                    </a>
+                    <a href="https://www.linkedin.com/company/pyopensci" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fab fa-linkedin mr-1"></i> LinkedIn
+                    </a>
+                    <a href="https://github.com/pyOpenSci" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fab fa-github mr-1"></i> GitHub
+                    </a>
+                    <a href="https://discord.gg/yYyDFP2BcP" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fa-brands fa-discord mr-1"></i> Discord
+                    </a>
+                    <a href="#" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fab fa-discourse mr-1"></i> Discourse
+                    </a>
+                    <a href="#" class="text-pyos-footer-text hover:text-pyos-deep-purple transition-colors" rel="nofollow noopener noreferrer">
+                        <i class="fas fa-rss mr-1"></i> Feed
+                    </a>
+                </div>
+            </div>
+            
+            <!-- Organizational Information -->
+            <div class="text-center space-y-4 text-sm text-pyos-footer-text mb-6 px-4">
+                <p>pyOpenSci is a fiscally sponsored project of <a href="https://communityinitiatives.org/" target="_blank" class="text-pyos-deep-purple hover:underline">Community Initiatives</a>, a registered 501(c)3 non-profit organization based in California, USA. pyOpenSci is grateful for support from the Sloan Foundation and the Chan Zuckerberg Initiative.</p>
+                <p>pyOpensci is dedicated to creating a welcoming, supportive and diverse community around the open source Python tools that drive open science. Our <a href="#" class="text-pyos-deep-purple hover:underline">Code of Conduct</a> defines expected behavior and guidelines that help create such a community.</p>
+            </div>
+            
+            <!-- Copyright -->
+            <div class="text-center text-sm text-pyos-footer-text">
+                <p>&copy; 2025 pyOpenSci</p>
+            </div>
+        </div>
+    </footer>
+    
+    <!-- Navigation JavaScript -->
+    <script src="{% static 'js/navigation.js' %}"></script>
+    
+    {% block extra_js %}{% endblock %}
+</body>
+>>>>>>> 40d013d (Fix mobile menu covering logo by adjusting position and z-index)
```

Comment:
This also looks like a remnant from a merge conflict. You will need to clean this up before we can review and merge it. Merge conflicts are always tricky but it's a great skill to learn how to deal with them using git!